sample/src/sidewalk_workitems.c

```diff
@@ -113,47 +109,53 @@ CREATE_WORKITEM(sidewalk_conn_request, {
 
 
 CREATE_WORKITEM(sidewalk_start, {
-    PRINT_SIDEWALK_VERSION();
-    app_ctx.is_init = false;
+    //PRINT_SIDEWALK_VERSION();
+    app_ctx->is_init = false;
     if (application_pal_init()) {
         LOG_ERR("Failed to initialze PAL layer for sidewalk applicaiton.");
         return;
     }
     sid_error_t err;
-    if((err = sidewalk_callbacks_set(&app_ctx, &app_ctx.event_callbacks)) != SID_ERROR_NONE){
+    if((err = sidewalk_callbacks_set(app_ctx, &app_ctx->event_callbacks)) != SID_ERROR_NONE){
```

Comment:
extracting this step outside the sidewal_start_handler would simplify the use of app_ctx, and you could than remove the global app_ctx pointer.